test/README.md

```diff
@@ -0,0 +1,6 @@
+ 
+This directory contains small executables to verify fluidsynths correct behaviour, i.e. unit tests. Although some tests might serve as educational demonstration of how to use certain parts of fluidsynth, they are **not** intended to do so! It is most likely that those tests will consist of many hacky parts that are necessary to test fluidsynth (e.g. including fluidsynth's private headers to access internal data types and functions). For user applications this programming style is strongly discouraged! Do **not** blindly use the tests as template for your application! Keep referring to the documentation and code examples listed in the [API documentation](http://www.fluidsynth.org/api/).
```

Comment:
Could you line wrap this file? Keeps it in line with all the other doc files we have (I think...) and makes life easier in general :-)